spec/presenters/curation_concerns/file_set_presenter_spec.rb

```diff
@@ -43,24 +43,22 @@
     it { is_expected.to eq 'File Set' }
   end
 
-  describe "date_uploaded" do
-    it "delegates to the solr_document" do
-      expect(solr_document).to receive(:date_uploaded)
-      presenter.date_uploaded
+  describe "properties delegated to solr_document" do
+    let(:solr_properties) do
+      # TODO: These properties are still missing
```

Comment:
I don't think this is helpful here. Can we just say "No" for now?  We've not demonstrated a need at this time.